doc/devel/license.rst

```diff
@@ -1,7 +1,7 @@
 .. _license-discussion:
 
-Licenses
-========
+Dependency licenses
```

Comment:
While one could argue about the definition of "dependency", the common use for it in the python context are other libraries that we depend upon but don't ship. The below text does not address these.

I agree that "Licenses" is not clear either. How about "Licenses for contributed code"?